### DISCUSSION TITLE
How do you set the slugs for assets and helpers?

### DISCUSSION THREAD

**Discussion by neil2ball** [GuduleLapointe/w4os#69] [Q&A] [answered]
How do you set the slugs for assets and helpers? I see some instructions, but nothing that is helping me find the web asset server or the helpers through WordPress.

**Comment by GuduleLapointe** [comment, marked as answer]
For web assets server, the URL is documented on Asset Server settings page, and there is a link to Permalinks option page to change the slug, after what the new correct URL would appear in Assets Server settings page.

For the helpers, it's less obvious. The base url is of the form http://www.yourgrid.org/helpers/, which is the URL for Economy helper, and other helpers URL use the same base (http://www.yourgrid.org/helpers/query.php for search, http://www.yourgrid.org/helpers/offline.php for online messages).

Remember
- use http:// and not https:// unless you know how to bypass Mono SSL certificates limitation (I can't help  you with that, it's a Mono issue)
- search requires the installation of third party OpenSimSearch dll
- ngynx require additional settings in ngynx config files

I think the helpers used to have a more obvious configurable slug, or should... I can't check it now, but I will have a look in the next days.

**Reply by neil2ball** [reply]
I can directly access the helpers using http://www.yourgrid.org/wp-content/plugins/w4os/helpers . Yet I don't see see them when I use http://www.yourgrid.org/helpers/. I get "That page can't be found."

**Reply by neil2ball** [reply]
I managed to locate assets in permalinks by enabling the curl extension in php.ini. I'm still working on helpers.

**Reply by GuduleLapointe** [reply]
I made the requirements clearer in the latest version (2.7.1 #c58975e).

> I managed to locate assets in permalinks by enabling the curl extension in php.ini. I'm still working on helpers.
